eng/testing/performance/performance-setup.sh

```diff
@@ -268,14 +293,18 @@ if [[ "$run_from_perf_repo" == true ]]; then
     performance_directory=$workitem_directory
     setup_arguments="--perf-hash $commit_sha $common_setup_arguments"
 else
-    git clone --branch main --depth 1 --quiet https://github.com/dotnet/performance.git $performance_directory
+    git clone --branch iosstartup --depth 1 https://github.com/akoeplinger/performance.git $performance_directory
```

Comment:
just curious, how long are we going to depend on @akoeplinger 's fork here? can we get this into dotnet/performance?

Reply:
Hopefully we won't depend on it at all. The matching PR is up, I just have not linked it yet. https://github.com/dotnet/performance/pull/2355 The PR should be merged and then the above code changed back before final merge. I have a mini TODO list at the top with that as one of the items.